src/actions/team/fetchTeam.tsx

```diff
@@ -0,0 +1,12 @@
+'use server'
+
+import { getTeamProfiles } from '@/backend/profile/profile.service'
+
+export async function fetchTeam() {
```

Comment:
**nit:** Actions are for POSTS, we don't have to keep them here, as you can just use `getTeamProfiles` directly in server component